src/Screens/Products/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './products.module.css';
+
+
+
+
+const Products = (props) => {
+  
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
```

Comment:
You shouldn't fetch data at the screen, that's why we use thunks and redux to store our information, remove this function and execute the corresponding thunk here